packages/table/components/Table.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from "react";
+import { css } from "emotion";
+import { AutoSizer, MultiGrid } from "react-virtualized";
+import { IColumnProps, Column } from "./Column";
+import memoizeOne from "memoize-one";
+import { coreColors as Colors } from "../../shared/styles/color";
+
+export interface ITableProps {
+  fixedColumnCount: number;
+  enableFixedColumnScroll: boolean;
+  enableFixedRowScroll: boolean;
```

Comment:
I don't think that we need to expose `fixedColumnCount`, `enableFixedColumnScroll`,  `enableFixedRowScroll` in most cases it will be `1`, `true` and `true`. At least I can't recall any example where we would need these props. Maybe let's not expose them now. to prevent later problems.